main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+
+        if player_data["race"].get("description") is None:
+            race = Race.objects.get_or_create(
+                name=player_data["race"]["name"]
+            )[0]
+        else:
+            race = Race.objects.get_or_create(
+                name=player_data["race"]["name"],
+                description=player_data["race"]["description"]
+            )[0]
```

Comment:
_description - a text field, can be blank._

This means that description can be blank when object creates


`Please simplify this to:`
```
race, _ = Race.objects.get_or_create(
            name=player_data["race"]["name"],
            description=player_data["race"]["description"]
)
```

Reply:
I didn't know who this method was, thank you, it's always nice to learn something new